components/automate-ui/src/app/page-components/projects-filter-dropdown/projects-filter-dropdown.component.ts

```diff
@@ -10,7 +10,7 @@ import { ProjectsFilterOption } from 'app/services/projects-filter/projects-filt
 export class ProjectsFilterDropdownComponent implements OnChanges {
   @Input() options: ProjectsFilterOption[] = [];
 
-  @Input() selectionLabel = 'All Resources';
+  @Input() selectionLabel = 'All projects';
```

Comment:
Is there a reason this doesn't use the constant?

Reply:
Yeah I've found this confusing as well. I think it's a default value only that doesn't ever get used. So, I've changed it for consistency, but it might not be the best fix. Not confident enough with this to remove all the defaults... :thinking_face:

Reply:
A. It would get used if the parameter is not supplied.
B. Please change it to use the constant; just overlooked.